PolyEngine/Engine/Src/ComponentBase.hpp

```diff
@@ -19,9 +29,13 @@ namespace Poly {
 			return Owner->GetComponent<T>();
 		}
 
+		void SetFlags(const EnumFlags<eComponentBaseFlags>& rhs) { Flags = rhs; }
+		void ClearFlags() { Flags = eComponentBaseFlags::NONE; }
```

Comment:
We can add Clear() method to EnumFlags. But clearing all flags for component is not usefull.